Orange/modelling/knn.py

```diff
@@ -0,0 +1,10 @@
+from Orange.classification import KNNLearner as KNNClassification
+from Orange.modelling.fitter import Fitter, LearnerTypes
+from Orange.regression import KNNRegressionLearner
+
+
+class KNNLearner(Fitter):
+    name = 'knn'
+
+    __fits__ = LearnerTypes(classification=KNNClassification,
+                            regression=KNNRegressionLearner)
```

Comment:
No, but what's the alternative? Should all the fitters be in `base.py`? That doesn't make too much sense either...